roles/terminal/tasks/zsh.yml

```diff
@@ -4,16 +4,16 @@
     apt:
         name: zsh
         state: present
--
-    name: set zsh as default shell to vagrant
-    user:
-        name: vagrant
-        shell: /bin/zsh
--
-    name: install oh-my-zsh
-    git:
-        repo: 'https://github.com/robbyrussell/oh-my-zsh.git'
-        depth: 1
-        dest: ~/.oh-my-zsh
-        version: master
-    become_user: "vagrant"
+    become: yes
```

Comment:
oh my zsh install instructions have changed a lot, so we use a community playbook here